__tests__/components/editor/FormWrapper.test.js

```diff
@@ -1,19 +1,22 @@
 // Copyright 2018 Stanford University see Apache2.txt for license
 import React from 'react'
 import { shallow } from 'enzyme'
-import PropertyTemplate from '../../../src/components/editor/PropertyTemplate'
+import InputLiteral from '../../../src/components/editor/InputLiteral'
```

Comment:
Let's keep PropertyTemplate for the types of properties we cannot yet do anything with (e.g. resource) ... eventually it will go away in favor of the specific treatments for each type.

Reply:
i've updated the code and added some header stubs that can be filled in by who ever is working on that type. for now its just a header saying " I am a ____type" where it can be filled in by `resource` `target` or `lookup`  - does that work for you?